Healthy/Classes/Modules/SavedRecipes/SavedRecipesViewController.swift

```diff
@@ -33,7 +48,42 @@ extension SavedRecipesViewController {}
 
 // MARK: - Configurations
 
-extension SavedRecipesViewController {}
+extension SavedRecipesViewController {
+    func configureNavigationBar() {
+        navigationBar.topItem?.title = "Saved recipes"
```

Comment:
```suggestion
        title = "Saved recipes"
```

Reply:
![Simulator Screen Shot - iPhone 14 Pro - 2023-07-06 at 02 47 33](https://github.com/motoon-eg/healthy/assets/45472327/dae15d9d-8b83-4d82-ade7-db728e515b42)
changes title of maintabbar not title of nib file